test/Ocelot.UnitTests/Security/IPSecurityPolicyTests.cs

```diff
@@ -335,57 +342,57 @@ private void GivenSetDownstreamRoute()
 
         private void GivenCidr24AllowedIP()
         {
-            _downstreamRouteBuilder.WithSecurityOptions(new SecurityOptions(new List<string> { "192.168.1.0/24" }, new List<string>(), false));
+            _downstreamRouteBuilder.WithSecurityOptionsCreator(new FileSecurityOptions { IPAllowedList = new List<string> { "192.168.1.0/24" }, IPBlockedList = new List<string>() });
```

Comment:
It is hard to read the structure of the object!
Too long sentence in one line!
Please provide formatting using multiline approach.

Reply:
Fixed code style